src/app/+run-tale/run-tale/tale-interact/tale-interact.component.html

```diff
@@ -13,14 +13,15 @@ <h3>Tale is launching, please wait...</h3>
         <iframe *ngIf="instance.iframe" height="100%" width="100%" [src]="instance.url | safe:'url'"></iframe>
 
         <div *ngIf="!instance.iframe" class="ui message">
-            This Tale must be run in a separate window.
-            <a class="ui button primary" [href]="instance.url" target="_blank">Click here to comply</a>
+            <p>This Tale must be run in a separate window.</p>
+            <p><a class="ui button primary" [href]="instance.url" target="_blank">Click here to comply</a></p>
```

Comment:
I just can't pass this opportunity...
```suggestion
            <p><a class="ui button primary"
                  [href]="instance.url"
                  [matTooltipShowDelay]="2000"
                  matTooltip="Take a deep breath. Calm your mind. You know what is best. What is best is you comply. Compliance will be rewarded."
                  target="_blank">
              Click here to comply</a>
            </p>
```

Reply:
Hmm.  Perhaps "Click here to open a new window" or simply "Click here" would suffice?

Reply:
No, cause that kills the easter egg... We are also discussing a page that won't be accessible to users, since all our images have `iframe=True`

Reply:
I do like Easter eggs like this :)

That said, if you don't get the reference then this is objectively creepy.